<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatchTestSettingsPage</class>
 <widget class="QWidget" name="CatchTestSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>323</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="showSuccessCB">
         <property name="toolTip">
          <string>Show success for tests.</string>
         </property>
         <property name="text">
          <string>Show success</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="breakOnFailCB">
         <property name="toolTip">
          <string>Turns failures into debugger breakpoints.</string>
         </property>
         <property name="text">
          <string>Break on failure while debugging</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="noThrowCB">
         <property name="toolTip">
          <string>Skips all assertions that test for thrown exceptions.</string>
         </property>
         <property name="text">
          <string>Skip throwing assertions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="visibleWhiteCB">
         <property name="toolTip">
          <string>Makes whitespace visible.</string>
         </property>
         <property name="text">
          <string>Visualize whitespace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="warnOnEmpty">
         <property name="toolTip">
          <string>Warns if a test section does not check any assertion.</string>
         </property>
         <property name="text">
          <string>Warn on empty tests</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="abortHL">
         <item>
          <widget class="QCheckBox" name="abortCB">
           <property name="toolTip">
            <string>Aborts after the specified number of failures.</string>
           </property>
           <property name="text">
            <string>Abort after</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="abortSB">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="samplesHL">
         <item>
          <widget class="QCheckBox" name="samplesCB">
           <property name="toolTip">
            <string>Number of samples to collect while running benchmarks.</string>
           </property>
           <property name="text">
            <string>Benchmark samples</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="samplesSB">
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="resamplesHL">
         <item>
          <widget class="QCheckBox" name="resamplesCB">
           <property name="toolTip">
            <string>Number of resamples used for statistical bootstrapping.</string>
           </property>
           <property name="text">
            <string>Benchmark resamples</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="resamplesSB">
           <property name="toolTip">
            <string>Number of resamples for bootstrapping.</string>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
           <property name="value">
            <number>100000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="confIntHL">
         <item>
          <widget class="QCheckBox" name="confIntCB">
           <property name="toolTip">
            <string>Confidence interval used for statistical bootstrapping.</string>
           </property>
           <property name="text">
            <string>Benchmark confidence interval</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="confIntSB">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="value">
            <double>0.950000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="warmupHL">
         <item>
          <widget class="QCheckBox" name="warmupCB">
           <property name="toolTip">
            <string>Warmup time for each test.</string>
           </property>
           <property name="text">
            <string>Benchmark warmup time</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="warmupSB">
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="noAnalysisCB">
         <property name="toolTip">
          <string>Disables statistical analysis and bootstrapping.</string>
         </property>
         <property name="text">
          <string>Disable analysis</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
